### PR TITLE
[RFR] Adding more Playbooks for testing.

### PR DESCRIPTION
**Pull request by sbulage**

### CODE REVIEW

**Comment by izapolsk** [comment]
@mshriver, are we going to store snippets or useful playbooks ?

File: set_retry_4_times.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+# This playbook is used to test the retry logic in a CloudForms state machine
```
Comment (mshriver) [inline set_retry_4_times.yml]: MIQ instead of CloudForms here please :)

File: timeout_60mins_playbook.yaml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+---
+- name: Do nothing for 60 minutes (or more)
```
Comment (mshriver) [inline timeout_60mins_playbook.yaml:2]: This seems awfully long for a test context, can it be shortened down to the lowest possible interval for the test? 5m? 1m even?

**Comment by mshriver** [comment]
@izapolsk its up to us - but I think either are appropriate for this repository, as it will be used to store automation playbooks for use in testing MIQ.

File: timeout_60mins_playbook.yaml (hunk 2 above)
Comment (sbulage) [reply timeout_60mins_playbook.yaml:2]: There was a customer BZ --> After running playbook for 60 mins or more, the time shown on the Service is something `01:65:02`.

**Comment by izapolsk** [comment]
> @izapolsk its up to us - but I think either are appropriate for this repository, as it will be used to store automation playbooks for use in testing MIQ.

I have the same in mind. I just would like to establish some basic rules and avoid having useless stuff in shared repo.

File: set_retry_4_times.yml (hunk 1 above)
Comment (sbulage) [reply set_retry_4_times.yml]: I have forked Devel repository, but I have updated it.

**Comment by sbulage** [comment]
@mshriver Updated it.

**Comment by sbulage** [comment]
@mshriver @izapolsk can be merged .?

**Review by mshriver — APPROVED** [review]

**Pull request merged**